commit 16a00368fb3a01001854040d2c29fae2d24e9642
Author: bertin <macportsuser@macports.org>
Date:   Fri May 23 21:28:46 2025 +0200

    /opt/local/site-ports/sysutils/MacPorts-pextlib/files/patch-sqlite3-from-MP.diff

diff --git a/Mk/macports.autoconf.mk.in b/Mk/macports.autoconf.mk.in
index 2b76c8c7..fee30f3c 100644
--- a/Mk/macports.autoconf.mk.in
+++ b/Mk/macports.autoconf.mk.in
@@ -27,7 +27,7 @@ TCL_STUB_LIB_SPEC	= @TCL_STUB_LIB_SPEC@
 LIBS			= @LIBS@
 READLINE_LIBS		= @READLINE_LIBS@
 MD5_LIBS		= @MD5_LIBS@
-SQLITE3_LIBS		= @abs_top_builddir@/@VENDOR_DESTROOT@@TCL_PREFIX@/lib/@SQLITE3_LIBNAME@/lib@SQLITE3_LIBNAME@$(SHLIB_SUFFIX)
+SQLITE3_LIBS	= @SQLITE3_LIBNAME@
 CURL_LIBS		= @LDFLAGS_LIBCURL@
 INSTALL			= @INSTALL@
 MTREE			= @MTREE@
diff --git a/configure.ac b/configure.ac
index d24b373f..c4241557 100644
--- a/configure.ac
+++ b/configure.ac
@@ -438,8 +438,10 @@ INCLUDES="-I.. -I. $INCLUDES"
 AC_SUBST(INCLUDES)
 
 ## sqlite3
-SQLITE3_CPPFLAGS=-I`echo ${TCL_SRC_DIR}/pkgs/sqlite3*/compat/sqlite3`
-SQLITE3_LIBNAME=`basename ${TCL_SRC_DIR}/pkgs/sqlite3*`
+# headers under $prefix/include as usual
+SQLITE3_CPPFLAGS=""
+# libsqlite3 will be in $prefix/lib; also business as usual
+SQLITE3_LIBNAME="-lsqlite3"
 AC_SUBST(SQLITE3_CPPFLAGS)
 AC_SUBST(SQLITE3_LIBNAME)
 
